t/react_template/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine AS webapp-builder
+WORKDIR /app
+COPY . .
+RUN apk add curl bash protoc
```

Comment:
We don't need curl or protoc for now, this template will only serve RESTful apis